### ISSUE TITLE
Are there any benchmark tests for this?

### ISSUE THREAD

**Issue by kevaundray** [albertorestifo/node-dijkstra#22]
I am looking to run this algorithm on one million nodes (A1.....An)

repeatedly doing route.path('A1'...'K')

So only the starting node changes, whereas the ending node will stay the same.

In spite of this, I would like to know whether any benchmarks have been performed regarding a large dataset?

**Comment by albertorestifo** [comment]
Nope. I simply did some local benchmarks when doing the rewrite to v2, to ensure performance improved.

Would be cool to have some as part of the test pipeline. Contributions welcomed!
